docs/vector.rst

```diff
@@ -101,6 +101,51 @@ As a temporary local variable
 
     TODO: add descriptions about ``a(i, j)``
 
+
+XYZW vector component accessors
++++++++++++++++++++++++++++++++
+
+We also provide four handy accessors for the first four vector components:
+
+.. attribute:: a.x
+
+   Same as ``a[0]``.
+
+.. attribute:: a.y
+
+   Same as ``a[1]``.
+
+.. attribute:: a.z
+
+   Same as ``a[2]``.
+
+.. attribute:: a.w
+
+   Same as ``a[3]``.
+
+.. note::
+
+   XYZW accessors can be used both read and write::
```

Comment:
```suggestion
   XYZW accessors can be used for both read and write:
```

Reply:
```suggestion
   XYZW accessors can be used for both read and write::
```
The `::` is placed intentionally so that the code below could be interpreted as `code`.